meta.yaml

```diff
@@ -5,6 +5,7 @@ OData-Data-Agg: https://docs.oasis-open.org/odata/odata-data-aggregation-ext/v4.
 OData-Temporal: https://docs.oasis-open.org/odata/odata-temporal-ext/v4.0/odata-temporal-ext-v4.0.html
 OData-Protocol: https://docs.oasis-open.org/odata/odata/v4.02/odata-v4.02-part1-protocol.html
 OData-URL: https://docs.oasis-open.org/odata/odata/v4.02/odata-v4.02-part2-url-conventions.html
```

Comment:
The `OData-URL` property will eventually be replaced by `odata-url-conventions`.

Reply:
Why? OData-URL is shorter and expressive enough.

Reply:
The new syntax uses the directory name to identify another spec document.